### ISSUE TITLE
Specify the datataset you want to use

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/lefser#3]
It will be clearer to specify the dataset you want, rather than using wildcards then `[[8]]` which add unnecessary complexity:

https://github.com/waldronlab/lefser/blob/ad7a229b02d1c947e7910e31efe5bd8b424d4f0c/vignettes/lefser.Rmd#L45

**Comment by lwaldron** [comment]
The currently selected dataset `[[8]]` can be retrieved as follows:
```R
dataset <- curatedMetagenomicData("DhakanDB_2019.metaphlan_bugs_list.stool", counts=TRUE, dryrun = FALSE)[[1]]
```
But the DhakanDB_2019 dataset is all controls and doesn't contain any adenoma patients. From the numerical argument `[[8]]` I'm not sure which dataset you wanted, and the order of datasets isn't guaranteed to remain constant with cMD updates. The above command downloads all 50-some metaphlan_bugs_list.stool datasets in cMD then selects one, so a great efficiency improvement will also be realized by just downloading the one dataset you want.

**Comment by lwaldron** [comment]
Solved in 15a6c93
